test/built-ins/Temporal/PlainDate/prototype/toPlainDateTime/limits.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.toplaindatetime
+description: Numeric bounds check
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const midnight = new Temporal.PlainTime(0, 0);
+const firstNs = new Temporal.PlainTime(0, 0, 0, 0, 0, 1);
+const lastNs = new Temporal.PlainTime(23, 59, 59, 999, 999, 999);
+const min = new Temporal.PlainDate(-271821, 4, 19);
+const max = new Temporal.PlainDate(275760, 9, 13);
+
+assert.throws(
+  RangeError,
+  () => min.toPlainDateTime(midnight),
+  'converting from Date and Time (1)'
+);
+
+assert.throws(
+  RangeError,
+  () => midnight.toPlainDateTime(min),
+  'converting from Date and Time (2)'
+);
```

Comment:
I think some of these tests (like this one) should go into a file in PlainTime/prototype/toPlainDateTime that mirrors this file.

Reply:
Quite right -- I thought this was the right place, but indeed some of the tests don't belong here. I've moved those to the right place.